From acfb12665eb042468b8356a6fbae9c5caf57f53d Mon Sep 17 00:00:00 2001
From: AKASHI Takahiro <takahiro.akashi@linaro.org>
Date: Mon, 20 Jul 2020 15:34:09 +0900
Subject: [PATCH 1/1] test/py: efi_secboot: remove unused function

'tool_is_in_path' function is no longer used anywhere after Heinrich
has removed 'sudo' version of fixture setup.

Signed-off-by: AKASHI Takahiro <takahiro.akashi@linaro.org>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/py/tests/test_efi_secboot/conftest.py | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/test/py/tests/test_efi_secboot/conftest.py b/test/py/tests/test_efi_secboot/conftest.py
index c6709700a8..4073732da6 100644
--- a/test/py/tests/test_efi_secboot/conftest.py
+++ b/test/py/tests/test_efi_secboot/conftest.py
@@ -8,15 +8,6 @@ from subprocess import call, check_call, check_output, CalledProcessError
 import pytest
 from defs import *
 
-# from test/py/conftest.py
-
-
-def tool_is_in_path(tool):
-    for path in os.environ["PATH"].split(os.pathsep):
-        full_path = os.path.join(path, tool)
-        if os.path.isfile(full_path) and os.access(full_path, os.X_OK):
-            return True
-    return False
 
 #
 # Fixture for UEFI secure boot test
-- 
2.27.0

